orm/query.go

```diff
@@ -295,7 +295,7 @@ func (q *Query) Column(columns ...string) *Query {
 			continue
 		}
 
-		//TODO: remove
+		// TODO: remove
```

Comment:
just FYI I believe this was correct - space is not used to distinguish from normal comments